CHANGELOG.md

```diff
@@ -9,6 +9,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+### Changed
+
+### Fixed
+
+[Unreleased]: https://github.com/SumoLogic/sumologic-kubernetes-collection/compare/v2.4.0-rc.0...main
+
+## [v2.4.0_rc.0][v2_4.0_rc.0]
+
+### Release v2.4.0-rc.0 - 2022-01-20
+
+### Added
```

Comment:
We have to figure out a better structure to this (how we deal with changelog) :)

Last release has those with `####` instead of `###`.

Reply:
That's fair point, but let's do it in a separate PR.